presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -178,53 +178,27 @@ constexpr folly::StringPiece kCounterMemoryCacheSumEvictScore{
 /// does not count.
 constexpr folly::StringPiece kCounterMemoryCacheNumCumulativeHit{
     "presto_cpp.memory_cache_num_cumulative_hit"};
-/// Number of hits (saved IO) since last counter retrieval. The first hit to a
-/// prefetched entry does not count.
-constexpr folly::StringPiece kCounterMemoryCacheNumHit{
-    "presto_cpp.memory_cache_num_hit"};
 /// Cumulated amount of hit bytes (saved IO). The first hit to a prefetched
 /// entry does not count.
 constexpr folly::StringPiece kCounterMemoryCacheCumulativeHitBytes{
     "presto_cpp.memory_cache_cumulative_hit_bytes"};
-/// Amount of hit bytes (saved IO) since last counter retrieval. The first hit
-/// to a prefetched entry does not count.
-constexpr folly::StringPiece kCounterMemoryCacheHitBytes{
-    "presto_cpp.memory_cache_hit_bytes"};
 /// Cumulated number of new entries created.
 constexpr folly::StringPiece kCounterMemoryCacheNumCumulativeNew{
     "presto_cpp.memory_cache_num_cumulative_new"};
-/// Number of new entries created since last counter retrieval.
-constexpr folly::StringPiece kCounterMemoryCacheNumNew{
-    "presto_cpp.memory_cache_num_new"};
 /// Cumulated number of times a valid entry was removed in order to make space.
 constexpr folly::StringPiece kCounterMemoryCacheNumCumulativeEvict{
     "presto_cpp.memory_cache_num_cumulative_evict"};
-/// Number of times a valid entry was removed in order to make space, since last
-/// counter retrieval.
-constexpr folly::StringPiece kCounterMemoryCacheNumEvict{
```

Comment:
Are we removing # of entries evicted counter? I thought it can be useful how eviction is working.

Reply:
Nope, still keep the memory_cache_num_cumulative_evict counter. Only removing the interval counters.